examples/quickstarts/helidon-quickstart-mp/pom.xml

```diff
@@ -113,6 +118,19 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.jboss.jandex</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>make-index</id>
+                        <goals>
+                            <goal>jandex</goal>
+                        </goals>
+                        <phase>process-classes</phase>
```

Comment:
This is the default phase for the plug-in. If you want to keep it here for clarity, fine. If you want to remove it for minimalism, fine.